### DISCUSSION TITLE
Custom sorting and "Folder"+Tag grouping of OTP accounts

### DISCUSSION THREAD

**Discussion by eugenesvk** [ente-io/ente#1255] [Enhancements]
Currently there is too much space wasted - you could fit way more codes on the screen to make selection without requiring search easier.

For the same reason it would be great if you could group your most used codes in "folders" or tag each code with any number of tags and then click on some tag to make it easier to find a code

Desktop Ente Auth app on Windows

**Comment by ua741** [comment]
Related discussion.

Allow custom sort order: https://github.com/ente-io/ente/discussions/497

Favorites & Folders: https://github.com/ente-io/ente/discussions/1039

Grid Layout https://github.com/ente-io/ente/discussions/1030
